test/unit/DCAStrategies/dca-strategies-permissions-handler.spec.ts

```diff
@@ -103,6 +104,52 @@ contract('DCAStrategiesPermissionsHandler', () => {
     });
   });
 
+  describe('set permissions', () => {
```

Comment:
You are already testing this in `mint`. I think you can do one of two things here:
1. Test this is `mint` only
2. Test this here only, and in `mint` check that `setPermissions` was called correctly

Reply:
I will go with option 2